doc/source/serve/production-guide/handling-dependencies.md

```diff
@@ -1,7 +1,29 @@
 (serve-handling-dependencies)=
 # Handle Dependencies
 
-Ray Serve supports serving deployments with different (possibly conflicting)
+## Adding a runtime environment
+
+The import path (e.g., `fruit:deployment_graph`) must be importable by Serve at runtime.
+When running locally, this might be in your current working directory.
+However, when running on a cluster you also need to make sure the path is importable.
+You can achieve this either by building the code into the cluster's container image (see [Cluster Configuration](kuberay-config) for more details) or by using a `runtime_env` with a [remote URI](remote-uris) that hosts the code in remote storage.
+
+As an example, we have [pushed a copy of the FruitStand deployment graph to GitHub](https://github.com/ray-project/test_dag/blob/40d61c141b9c37853a7014b8659fc7f23c1d04f6/fruit.py). You can use this config file to deploy the `FruitStand` deployment graph to your own Ray cluster even if you don't have the code locally:
```

Comment:
Did my best to understand the relationship between the deployment graph and the config file. If I got it wrong, I suggest you clarify the transition from the fruitstand deployment graph, to using "this" config file. Are they the same thing? Is one related to the other?

Reply:
Hm this is focused more on the runtime env part. The takeaway should be - if you're deploying on a remote cluster you need to put your code either in the runtime environment which uses a remote URI or in the cluster image itself. Do you think that comes out clearly?

Reply:
Yes, that part comes through. I was just commenting on the structure of the last sentence. If you think it will be clear to the average dev, then let's close this convo.